src/tools/ceph_monstore_tool.cc

```diff
@@ -655,6 +655,14 @@ static int update_pgmap_meta(MonitorDBStore& st)
     t->put(prefix, "full_ratio", bl);
   }
   {
+    auto backfillfull_ratio = g_ceph_context->_conf->mon_osd_backfillfull_ratio;
+    if (backfillfull_ratio > 1.0)
+      backfillfull_ratio /= 100.0;
```

Comment:
Yes, I did this like was done elsewhere.  If someone puts in 80 instead .80, it works just as well.